index.html

```diff
@@ -24,17 +24,127 @@ <h1 class="main_content">
+                    <ul class="cate">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button class="button-flex">See Project</button>
+                </div>
+                <div class="grid-item">
+                    <div class="img-holder"></div>
+                    <h3 class="text3">Multi-Post Stories Gain+Glory</h3>
+                    <ul class="cate">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button class="button-flex">See Project</button>
+                </div>
+                <div class="grid-item">
+                    <div class="img-holder"></div>
+                    <h3 class="text3">Multi-Post Stories Gain+Glory</h3>
+                    <ul class="cate">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button class="button-flex">See Project</button>
+                </div>
+             </div>
+            </section>
+            <section class="about-me-section">
+                    
+                <div class="about-me">
+                    <img class="il-about il-1-a" src="illustration 1- about me-mobile.svg">
+                     <img class="il-about il-2-a" src="illustration 2-about me mobile.svg">
+                    <h2 class="h2-about">About me</h2>
+                    <p class="p-about"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don't hestiate to contact me.</p>
+                     <button class="button-about">Get My Resume</button>
+             </section>
+                    </div>
+                <div class="about-me-grid">
+                        <div class="grid-item-about">
+                            <div class="img-holder-about"><img src="about-me-images/icon-languages.svg"></div>
+                            <h3 class="text3-about">Languages</h3>
+                            <ul class="list-about">
+                                <li>HTML</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                            </ul>
+                        </div>
+                        <div class="grid-item-about">
+                            <div class="img-holder-about"><img src="about-me-images/Icons.svg"></div>
+                            <h3 class="text3-about">Frameworks</h3>
+                            <ul class="list-about">
+                                <li>HTML</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
```

Comment:
- [ ] Kindly make sure to add the `border-radius` on each technology background. in order to look like the provided Figma design. :wink: 

  - ### YOUR DESIGN
  
    ![image](https://user-images.githubusercontent.com/87197412/212335576-fe8baeea-fd07-4ee2-9f68-1e482d6d9e15.png)

  - ### FIGMA DESIGN
  
     ![image](https://user-images.githubusercontent.com/87197412/212335848-ffb63adc-d422-4605-84c4-ebcd6a4f0948.png)